extensions/KHR/EGL_KHR_platform_wayland.txt

```diff
@@ -93,6 +94,16 @@ New Behavior
     until the next buffer swap. The rationale behind this behavior is to keep
     operations result accurate until the next swap.
 
+    During the call to eglSwapBuffers wl_surface.attach will be requested with
+    the new front buffer, a full surface damage is requested via
+    wl_surface.damage or wl_surface.damage_buffer, and a wl_surface.commit will
+    be requested to commit the new state.
```

Comment:
To me this language already implies that, but you could say something like "Before the call to eglSwapBuffers() returns" or similar, rather than "During the call to." It could still happen in another thread, but has to happen before the call ends from the calling thread's perspective, rather than some mangled interpretation of the call's duration from the driver's point of view.

Reply:
Is that better? I left out mentioning threads, because I guess it's still technically possible to offload anything to a thread, as long as the thread calling eglSwapBuffers is blocked.

Reply:
I like it. Thanks!